tests/code/test_term.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+
+from app.models import mainDB
+from app.models.term import Term
+from app.logic.utils import selectSurroundingTerms
+from app.logic.term import addNextTerm, changeCurrentTerm, addPastTerm
+
+
+
+@pytest.mark.integration
+def test_selectSurroundingTerms():
+    listOfTerms = selectSurroundingTerms(Term.get_by_id(3))
+    assert 8 == len(listOfTerms)
+
+    listOfTerms = selectSurroundingTerms(Term.get_by_id(3), prevTerms=0)
+    assert [3,4,5,6,7,8] == [t.id for t in listOfTerms]
+
+    listOfTerms = selectSurroundingTerms(Term.get_by_id(3), prevTerms=1)
+    assert [2,3,4,5,6,7,8] == [t.id for t in listOfTerms]
+
+    listOfTerms = selectSurroundingTerms(Term.get_by_id(3), prevTerms=-1)
+    assert [4,5,6,7,8] == [t.id for t in listOfTerms]
+
+def test_changeCurrentTerm():
+    # test via g.current_term
+    oldTerm = g.current_term
+    changeCurrentTerm(2)
+    assert g.current_term == 2
+    assert not oldTerm
+
+    # test via isCurrentTerm
+    oldTerm2 = g.current_term
+    newTerm = changeCurrentTerm(1)
+    assert newTerm.isCurrentTerm
+    assert not oldTerm2 == g.current_term
+    assert not oldTerm2.isCurrentTerm
+
+    # reset data back to before test
+    changeCurrentTerm(oldTerm)
+
+def test_invalidTermInputs():
+    with pytest.raises(DoesNotExist):
+        changeCurrentTerm(100)
+    with pytest.raises(DoesNotExist):
+        changeCurrentTerm("womp")
+
+@pytest.mark.integration
+def test_addNextTerm():
+    with mainDB.atomic() as transaction:
+        testTerm = Term.create(description="Summer 2022",year=2022, academicYear= "2021-2022", isSummer=True, isCurrentTerm=True, termOrder = Term.convertTerm("Summer 2022"))
+        testTerm.save()
+
+        addNextTerm()
+
+        # for the first test,  make sure we're using the db properly
```

Comment:
If you want to keep this comment please make it more descriptive. What do you mean you are making sure you are using the db properly? Also, this comment gives the impression that you will have all of the following tests in this section commented as well but you don't.